From: Mathieu Parent <math.parent@gmail.com>
Date: Mon, 31 Oct 2016 20:45:27 +0100
Subject: nmbd requires a working network

Bug-Debian: https://bugs.debian.org/698056
Bug-Debian: https://bugs.debian.org/842056
Bug-Debian: https://bugs.debian.org/840608
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/samba/+bug/1635491

---
 packaging/systemd/nmb.service | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/packaging/systemd/nmb.service b/packaging/systemd/nmb.service
index c02e65d..71cf1c2 100644
--- a/packaging/systemd/nmb.service
+++ b/packaging/systemd/nmb.service
@@ -1,7 +1,8 @@
 [Unit]
 Description=Samba NMB Daemon
 Documentation=man:nmbd(8) man:samba(7) man:smb.conf(5)
-After=network.target
+After=network-online.target
+Wants=network-online.target
 
 [Service]
 Type=notify
-- 
2.9.3

