From: Jelmer Vernooĳ <jelmer@jelmer.uk>
Date: Sat, 9 Apr 2016 12:18:38 +0000
Subject: drop host-specific define that prevents reproducible builds

Index: samba/buildtools/wafsamba/wscript
===================================================================
--- samba.orig/buildtools/wafsamba/wscript
+++ samba/buildtools/wafsamba/wscript
@@ -303,7 +303,6 @@ def configure(conf):
 
     conf.CHECK_COMMAND(['uname', '-a'],
                        msg='Checking build system',
-                       define='BUILD_SYSTEM',
                        on_target=False)
     conf.CHECK_UNAME()
 
