From: Jelmer Vernooĳ <jelmer@jelmer.uk>
Date: Sat, 9 Apr 2016 12:18:38 +0000
Subject: drop host-specific define that prevents reproducible builds

---
 buildtools/wafsamba/wscript | 1 -
 1 file changed, 1 deletion(-)

diff --git a/buildtools/wafsamba/wscript b/buildtools/wafsamba/wscript
index 430d164..8c82b25 100644
--- a/buildtools/wafsamba/wscript
+++ b/buildtools/wafsamba/wscript
@@ -315,7 +315,6 @@ def configure(conf):
 
     conf.CHECK_COMMAND(['uname', '-a'],
                        msg='Checking build system',
-                       define='BUILD_SYSTEM',
                        on_target=False)
     conf.CHECK_UNAME()
 
